File: app/models/dummy/Archive.java
package models.dummy;

import models.IArchive;

/**
 * Created by Beemen on 30/07/2015.
 */
public class Archive implements IArchive {

    private String _ReferenceCode;
    @Override
    public String ReferenceCode() {
        return _ReferenceCode;
    }

    @Override
    public void ReferenceCode(String value) {
        this._ReferenceCode = value;
    }

    private String _AirUrl;
    @Override
    public String AipUri() {
        return _AirUrl;
    }

    @Override
    public void AipUri(String value) {
        _AirUrl = value;
    }
}

File: Models/dummy/Archive.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace end_user_gui.Models.dummy
{
    public class Archive : IArchive
    {
        public String ReferenceCode { get; set; }
        public String AipUri { get; set; }

        List<IArchiveFile> _Files = new List<IArchiveFile>();
        public List<IArchiveFile> Files
        {
            get { return _Files; }
            set { _Files = value; }
        }
    }
}
